dev/tasks/r/github.linux.offline.build.yml

```diff
@@ -84,7 +84,7 @@ jobs:
         {{ macros.github_set_sccache_envvars()|indent(8)}}
         run: |
           cd arrow/r
-          R CMD INSTALL --install-tests --no-test-load --no-docs --no-help --no-byte-compile arrow_with_deps.tar.gz
+          R CMD INSTALL --install-tests --no-test-load --no-byte-compile arrow_with_deps.tar.gz
```

Comment:
Apparently docs are checked in 4.4 regardless and we get an error about corrupt docs. It's not too much extra time to build them though